README.md

```diff
@@ -7,9 +7,13 @@ For a more complete description of the model, see:
 
 [Chisholm, Robert, Paul Richmond, and Steve Maddock. "A standardised benchmark for assessing the performance of fixed radius near neighbours." European Conference on Parallel Processing. Springer, Cham, 2016.](https://doi.org/10.1007/978-3-319-58943-5_25), ([pdf](https://eprints.whiterose.ac.uk/104079/1/paper.pdf)).
 
-## Benchmark Results
+## Benchmark Description and Results
 
-Two experiments are carried out:
+Three experiments are undertaken within this benchmark. There is a range of raw data in the [`sample/data`](sample/data) directory with a description of the machine configurations used to generate it in each directory.
+
+The results below are from the V100 runs on the Bessemer HPC system at the University of Sheffield. Job submission scripts are included in the [`scripts/slurm`]() folder.
+
+### Fixed Density Benchmark
 
 + Fixed Density
```

Comment:
This top level list item is redundant. 

Unfortunately I can't get multi-line suggestions to work here, but the top item can be removed and subsequent list items deindented.